CHANGELOG.md

```diff
@@ -76,6 +76,8 @@ Compatibility:
 * Fix `Enumerable` methods `each_cons` and `each_slice` to return receiver (#2733, @horakivo)
 * `Module` methods `#private`, `#public`, `#protected`, `#module_function` now returns their arguments like in CRuby 3.1 (#2733, @horakivo)
 * `Kernel#exit!`, killing Fibers and internal errors do not run code in `ensure` clauses anymore, the same as CRuby (@eregon).
+* Implement `UnboundMethod#original_name` (@paracycle, @nirvdrum).
+* Replaced the non-standard, private `UnboundMethod#origin` with a primitive (@paracycle, @nirvdrum).
```

Comment:
Since this is not really user-visible (well, unless they called a private method) I'm not sure it should be in the CHANGELOG

Reply:
That's fine. We can remove it.